### ISSUE TITLE
[3] Accessibility issues: Detect UI elements that are missing descriptions

### ISSUE THREAD

**Issue by connected-cziesman** [connected-io/UnIt#50]
Detect UI elements that are missing meaningful Accessibility descriptions

**Comment by connected-cziesman** [comment]
Not sure if this makes sense to do. The real way to find accessibility issues is to use Accessibility Inspector, which can find many different types of issues. It might be helpful to return an array of subviews that have nil (default) for accessibility traits, values, hints, etc (everything) to indicate views that were implemented with no accessibility in mind... or indicate a previously accessible view that got replaced with something that hasnt had any accessibility implemented yet.
